examples/plot_simulate_alpha.py

```diff
@@ -42,24 +50,71 @@
 # simulation. Each burst consists of a pair (2) of spikes, spaced 10 ms apart.
 # The occurrence of each burst is jittered by a random, normally distributed
 # amount (20 ms standard deviation). We repeat the burst train 10 times, each
-# time with unique randomization. The drive is only connected to the distal
+# time with unique randomization. The drive is only connected to the proximal
 # (dendritic) AMPA synapses on L2/3 and L5 pyramidal neurons.
 net = Network(params)
 
-weights_ampa = {'L2_pyramidal': 5.4e-5, 'L5_pyramidal': 5.4e-5}
+location = 'proximal'
+burst_std = 20
+weights_ampa_p = {'L2_pyramidal': 5.4e-5, 'L5_pyramidal': 5.4e-5}
+syn_delays_p = {'L2_basket': 0.1, 'L2_pyramidal': 0.1,
+                'L5_basket': 1., 'L5_pyramidal': 1.}
+
 net.add_bursty_drive(
-    'bursty', tstart=50., burst_rate=10, burst_std=20., numspikes=2,
-    spike_isi=10, repeats=10, location='distal', weights_ampa=weights_ampa,
-    seedcore=4)
+    'alpha_prox', tstart=50., burst_rate=10, burst_std=burst_std, numspikes=2,
+    spike_isi=10, repeats=10, location=location, weights_ampa=weights_ampa_p,
+    synaptic_delays=syn_delays_p, seedcore=13)
 
 dpl = simulate_dipole(net)
 
 trial_idx = 0  # single trial simulated
 dpl[trial_idx].plot()
 
 ###############################################################################
-# We can confirm that what we simulate is indeed 10 Hz activity.
+# We can confirm that what we simulate is indeed 10 Hz activity by plotting the
+# power spectral density. Note that the SciPy-function
+# `~scipy.signal.spectrogram` is used to create the plot.
 import matplotlib.pyplot as plt
 from hnn_core.viz import plot_spectrogram
 tmin = 20  # exclude initial burn-in period
 plot_spectrogram(dpl[trial_idx], fmin=0., fmax=40., tmin=tmin)
+
+###############################################################################
+# The next step is to add a simultaneous 10 Hz distal drive. Due to the
+# stochasticity of input spike timing, the proximal and distal spikes
+# occasionally arrive at the same time which will result in a beta frequency
+# (15-30 Hz) event.
```

Comment:
I don't quite understand why "proximal and distal drives" arriving at the same time results in beta frequency. Maybe more explanation needed for computer scientists? :)

Reply:
You’re right: it’s actually related to the interaction between the two opposing dipole orientations, neuron morphology and (in some cases) a difference between the std of the two bursts! The Law2019-paper my demo cites shows this pretty clearly. We should re-phrase this...

Reply:
Added a brief note that this arises from the biophysical properties of neurons.